component.json

```diff
@@ -255,20 +255,37 @@
       }
     },
     "delete": {
-      "title": "Delete Object",
+      "title": "Delete Object (at most 1)",
       "main": "./lib/actions/deleteObject.js",
       "description": "Delete Selected Object",
+      "dynamicMetadata": true,
```

Comment:
If you use dynamic metadata, then you should remove the `metadata` section for the action (as both parts will now come from the dynamic metadata.